src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -407,6 +407,11 @@ export const LandscapePageWrapper=styled.div`
 	.react-tabs__tab {
 		bottom: 0px;
 		padding: 1.25rem;
+		background-color: ${props => props.theme.primaryColor};
+		border: 1px solid ${props => props.theme.shadowLightColor};
+		&:nth-child(1){
+			border-top-left-radius: .5rem;
+		}
 		&:hover {
 			background-color: ${props => props.theme.tertiaryColor};
 			border-top-left-radius: .5rem;
```

Comment:
Update this to be similar to the above code, i.e, only for `nth-child(1)` and the `border-left radius` should only be updated

Reply:
Also, update the code at line: 432, `.react-tabs__tab--selected`, to modify the `border-radius` to `0`.

Reply:
> 

I have not understood that point

Reply:
`&:nth-child(1){
			border-left-radius: .5rem;
		}`

do mean like that?

Reply:
But ig in css there is no `border-left-radius` porperty....I have used that `border-top-left-radius` for matching with main table corner (border-top-right-radius of the main table)

Reply:
Yes, I know. What I meant was, update the CSS like:
```
.react-tabs__tab {
		bottom: 0px;
		padding: 1.25rem;
		background-color: ${props => props.theme.primaryColor};
		border: 1px solid ${props => props.theme.shadowLightColor};
		&:nth-child(1){
			border-top-left-radius: .5rem;
		}
		&:hover {
			background-color: ${props => props.theme.tertiaryColor};
                        &:nth-child(1){                                               |       <-- This was the change I was referring to
			        border-top-left-radius: .5rem;                |
		        }                                                                     |
                        ......
                        .......
```

Reply:
Ok got under stood...

Reply:
But even after putting this code...there is no change in the hover effect..it is similar as before

Reply:
&:hover {
			background-color: ${props => props.theme.tertiaryColor};
			&:nth-child(1){
				border-top-left-radius: .5rem;
			}
			border-top-left-radius: .5rem;
			border-top-right-radius: .5rem;
		}

Reply:
Only change that is when we secreting a tab it like that: (after modifying the `border-radius` to `0` of  `.react-tabs__tab--selected`

Reply:
> &:hover { background-color: ${props => props.theme.tertiaryColor}; &:nth-child(1){ border-top-left-radius: .5rem; } border-top-left-radius: .5rem; border-top-right-radius: .5rem; }

Remove the last two lines, we don't need that. As, the radius should only be added for the 1st option.
Reason being, if you notice in the current preview because of radius the corners of the tab show the green color of the background.

Reply:
Yes, I am also thinking that...it is looking weird...But it is from the previous UI.. I have not changed that...so let me change it now

Reply:
Any other changes are required?

Reply:
None, that I can think of now.

Reply:
Ok..